package.json

```
@@ -19,7 +19,8 @@
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
     "lint-api": "oval validate -p openapi/index.yaml",
     "generate": "npm-run-all generate:*",
-    "generate:definitions": "rimraf ./generated/definitions && shx mkdir -p ./generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --no-strict --out-dir ./generated/definitions",
+    "generate:definitions-internal": "rimraf ./generated/definitions/internal && shx mkdir -p ./generated/definitions/internal && gen-api-models --api-spec ./openapi/internal.yaml --no-strict --out-dir ./generated/definitions/internal ",
+    "generate:definitions-external": "rimraf ./generated/definitions/external && shx mkdir -p ./generated/definitions/external && gen-api-models --api-spec ./openapi/external.yaml --no-strict --out-dir ./generated/definitions/external --request-types --response-decoders --client",
```

We don't need to generate client, since this APIs are exposed by the component itself
```suggestion
    "generate:definitions-internal": "rimraf ./generated/definitions/internal && shx mkdir -p ./generated/definitions/internal && gen-api-models --api-spec ./openapi/internal.yaml --no-strict --out-dir ./generated/definitions/internal",
    "generate:definitions-external": "rimraf ./generated/definitions/external && shx mkdir -p ./generated/definitions/external && gen-api-models --api-spec ./openapi/external.yaml --no-strict --out-dir ./generated/definitions/external",
```